DMTN-102.tex

```diff
@@ -112,83 +114,86 @@ \subsection{Alert Packet Size}\label{ssec:packet_size}
+% As described in Section 2.2.3 of \citeds{LDM-612}: {\it "An allocation of 10~Gbps is baselined for alert stream transfer from the [LSST Data Facility], with an estimated packet size of 82~KB and up to 10,000 alerts per visit. For illustration, based on these numbers up to 7 brokers could receive the full stream if 5 seconds is budgeted for outbound data transfer."} 
 
 
 % % % % % % % % % % % % %
 \subsection{Alert Database Volume}\label{ssec:adb_volume}
 
 {\bf The estimated size of the alerts database after 10 years is $\sim$2.2~PB.}
 
-There are no formal requirements on the alerts database volume. The statement above is an estimate based on the alert packet contents, the number of alerts per night, and the expected number of observing nights per year. As described in \S~\ref{ssec:transN}, the DMS system will support an average of $\sim$10 million alerts per night (which approximately matches the expected scientific yields). Assuming the upper estimate of $\sim$82~KB per alert (\S~\ref{ssec:packet_size}), that leads to a total of $\sim$782~GB per night. After accounting for downtime and weather\ossreq{0080}\ossreq{0081}\ossreq{0082} the total number of observing nights is 300 per year \citedsp{LSE-30}, which leads to an estimate of $\sim$2.2~PB after 10 years. Compression could drastically lower this, as could reformatting: every alert contains a $\sim$12 month historical record and links to the most recent DIAObject and DR Object catalogs. The set of alerts for the same transient/variable would contain a significant amount of redundant information, which is provided to enable rapid filtering but could be reformatted for long-term storage (i.e., removed from the individual alerts and compiled elsewhere in an alerts database). 
+There are no requirements on the alerts database volume. The statement above is an estimate based on the alert packet contents, the number of alerts per night, and the expected number of observing nights per year. As described in \S~\ref{ssec:transN}, the DMS system will support an average of $\sim$10 million alerts per night (which approximately matches the expected scientific yields). Assuming the upper estimate of $\sim$82~KB per alert (\S~\ref{ssec:packet_size}), that leads to a total of $\sim$782~GB per night. After accounting for downtime and weather\ossreq{0080}\ossreq{0081}\ossreq{0082} the total number of observing nights is 300 per year \citedsp{LSE-30}, which leads to an estimate of $\sim$2.2~PB after 10 years. Compression could drastically lower this, as could reformatting: every alert contains a $\sim$12 month historical record and links to the most recent DIAObject and DR Object catalogs. The set of alerts for the same transient/variable would contain a significant amount of redundant information, which is provided to enable rapid filtering but could be reformatted for long-term storage (i.e., removed from the individual alerts and compiled elsewhere in an alerts database). 
```

Comment:
I'd remove the last parenthetical--the alerts database is meant to be a literal copy of the alerts as transmitted.  The PPDB is the non-redundant version of that information.

Reply:
100%.  I think we shouldn't even mention "reformatting" here.  It would be a major LCR to decide that the alerts database be changed to not be a verbatim transcript of the transmissions.

Reply:
Deleted everything in the statement "Compression could ... in an alerts database)."
Though it's true we're considering options for alerts, I'll take the stance that since nothing is yet decided there's no need to complicate this volume estimate with maybes.
We would just update DMTN-102 if this changes.